lang/fr.json

```diff
@@ -310,8 +310,8 @@
       }
     },
     "connStatus": {
-      "0": "Déconnecté",
-      "1": "Connecté",
+      "0": "Déconnecté(e)",
+      "1": "Connecté(e)",
```

Comment:
What's the reason for the parentheses? It should only ever represent one state.

Reply:
In french, adjectives like connecté differs if the person is male or female. In this case, Connecté was only for male, while Connecté(e) means that the person is or male or female.